### PR TITLE
duplicated of json field

### PR DESCRIPTION
**Pull request by appleboy**
see https://github.com/go-gitea/go-sdk/blob/master/gitea/release.go#L17-L19

### CODE REVIEW

**Comment by tboerger** [comment]
LGTM

**Comment by lunny** [comment]
Is the change compatible with Github's API?

**Comment by tboerger** [comment]
> Is the change compatible with Github's API?

Just look some lines above, the JSON field is already taken by the tag name.

**Comment by sapk** [comment]
As described here https://developer.github.com/v3/repos/releases/. It is TagName that should be rename and not title.

File: gitea/release.go
Comment (sapk) [inline gitea/release.go]: should be "tag_name"

File: gitea/release.go
Comment (sapk) [inline gitea/release.go]: keep "name" here

**Review by sapk — CHANGES_REQUESTED** [review]
As described here https://developer.github.com/v3/repos/releases/. It is TagName that should be rename and not title.

**Comment by appleboy** [comment]
@sapk @lunny @tboerger updated.

**Comment by sapk** [comment]
First LGTM ^^

**Comment by strk** [comment]
On Sat, Apr 15, 2017 at 08:49:12AM -0700, Antoine GIRARD wrote:
> First LGTM ^^

Does it mean  you're a maintainer now ? Because I don't see you
yet in the MAINTAINERS file ?

Take my LGTM if yours isn't good (I'm trusting you)

**Comment by appleboy** [comment]
@sapk Could you update this https://github.com/go-gitea/go-sdk/blob/master/MAINTAINERS file in all repository of Gitea organization？

**Pull request merged**
